extensions/vscode/src/granite/panels/setupGranitePage.ts

```diff
@@ -1,32 +1,32 @@
 import { LocalModelSize } from "core";
 import { ConfigHandler } from "core/config/ConfigHandler";
 import {
-  DEFAULT_MODEL_GRANITE_LARGE,
-  DEFAULT_MODEL_GRANITE_SMALL,
+    DEFAULT_MODEL_GRANITE_LARGE,
+    DEFAULT_MODEL_GRANITE_SMALL,
```

Comment:
Hmm, why the spaces added here? Did your identation settings get confused somehow?

Reply:
no idea what happened. It fixed itself next time I saved.